tsconfig.json

```diff
@@ -20,7 +21,11 @@
     ],
     "typeRoots": [
       "node_modules/@types"
-    ]
+    ],
+    "paths": {
+      "@core": ["src/core"],
```

Comment:
Would it be better to have only one path like?
```
"@sdk": ["src"],
```

The index.ts are automatically generated, no subpaths are required.

Reply:
yeah, agree, 
we need to add imports to the index.ts files

Reply:
We should add  "prettier-plugin-organize-imports" and "create-ts-index" to automatically generate index files and sort imports. 

What about calling the paths:

"symbol-sdk": ["src"], (assuming the npm package will be call symbol-sdk, could be something else) so the imports look exactly like an app will import modules.

Reply:
not sure `symbol-sdk` will be a good idea.
Because we name the folder as `nis1` and `symbol`.

should we rename the folder `symbol` -> `catapult` ?

Reply:
> We should add "prettier-plugin-organize-imports" and "create-ts-index" to automatically generate index files and sort imports.
> 
> What about calling the paths:
> 
> "symbol-sdk": ["src"], (assuming the npm package will be call symbol-sdk, could be something else) so the imports look exactly like an app will import modules.

not sure we need this, especially the `create-ts-index`, this package is not very well maintained and throws lots of security alerts from npm

Reply:
It's  a petty about create-ts-index not being maintain, thinking about forking it myself.

`prettier-plugin-organize-imports` is still useful to get the imports sorted.